### ISSUE TITLE
Realtek Audio Device Tweak problem....

### ISSUE THREAD

**Issue by xenon205** [AlanFinotty1995/AAFRealtekMod#10]
Hi... i wanted to check what Realtek Audio Device Tweak does. It said "test utility" and thought i won't brake anything with it... but i did.

(ErrorCode 1) <= HDAUDIO\FUNC_01&VEN_10EC&DEV_0892&SUBSYS_15586B01&REV_1003\4&35F46B4&0&0001
Power Data: Bus(D3), Codec(D3)

Any idea how to fix it :(

**Comment by AlanFinotty1995** [comment]
Have you made any changes to this tool ?

**Comment by xenon205** [comment]
Yes i did. I fixed it by my self but just maybe there should be a warning about it :)
There is one thing that just maybe i could fix with this app. When i plug in 3 jacks from my 5.1 set, headphone /  front speaker port stop responding and only power down fixes it. When i played with this app i noticed that this problem dissapired but now it's back again. Is there any thing that comes to your mind that could fix my problem?
